R/guides-.r

```diff
@@ -263,7 +263,7 @@ guides_build <- function(ggrobs, theme) {
     # add space between the guide-boxes
     guides <- gtable_add_col_space(guides, theme$legend.spacing.x)
 
-  } else if (theme$legend.box == "vertical") {
+  } else { # theme$legend.box == "vertical"
```

Comment:
Can you change the test in the first block to use `identical()`? That's safer if `theme$legend.box` is somehow not a length-1 vector. (I'm surprised the user of `NULL` doesn't cause a warning, TBH)

Reply:
Done.